### PR TITLE
test: Adds tests to the AdvancedFrame component

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Adds tests to the `AdvancedFrame` component.

### TEST PLAN
1 - Execute `AdvancedFrame` tests.
2 - All tests should pass.

@rusackas @junlincc 

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/explore/components/controls/DateFilterControl/components/AdvancedFrame.test.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import userEvent from '@testing-library/user-event';
+import { AdvancedFrame } from 'src/explore/components/controls/DateFilterControl/components/AdvancedFrame';
```
Comment (geido) [inline superset-frontend/src/explore/components/controls/DateFilterControl/components/AdvancedFrame.test.tsx:22]: A relative path here might do just fine?

**Review by geido — APPROVED** [review]
LGTM!

File: superset-frontend/src/explore/components/controls/DateFilterControl/components/AdvancedFrame.test.tsx (hunk 1 above)
Comment (michael-s-molina) [reply superset-frontend/src/explore/components/controls/DateFilterControl/components/AdvancedFrame.test.tsx]: Fixed.

**Review by rusackas — APPROVED** [review]
LGTM!

**Pull request merged**